cucumber/authenticators/features/step_definitions/authn_common_steps.rb

```diff
@@ -17,3 +17,11 @@
 Then(/it is forbidden/) do
   expect(forbidden?).to be true
 end
+
+Then(/it is read timeout/) do
```

Comment:
what is "read timeout"?

Reply:
The scenario here is the client (cucumber) gets the timeout before conjur (still stuck to resolve unreachable hostname), so the client gets read timeout error and null in the http returned code, so this is the way to validate the client gets timeout error